classes/blockchain-stats.class.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * BlockchainStats
+ *
+ * A PHP class for statistics about the bitcoin blockchain.
+ *
+ * @author     bjadel
+ * @version    0.1.0
+ */
+class BlockchainStats {
+
+  public function __construct() {
+  }
+
+  public static function satToFiat($sats, $fiat="EUR") {
+
+    $curl = curl_init();
+    $url = "https://blockchain.info/ticker";
+    curl_setopt($curl, CURLOPT_URL, $url);
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+    $result = curl_exec($curl);
+    $result = json_decode($result, true);
+    $bitcoin_rate = $result[$fiat]["15m"];
+    $result = $bitcoin_rate * ($sats/100000000);
+
+    curl_close($curl);
+
+    return $result;
+}
```

Comment:
Can we update this to use WP_Http::request instead of curl? This is reccommended to pass WordPress plugin review.

Reply:
> Can we update this to use WP_Http::request instead of curl? This is reccommended to pass WordPress plugin review.

Do you mean a call like this: $request = WP_REST_Request::from_url( 'http://example.com/wp-json/wp/v2/posts' );